### DISCUSSION TITLE
Probably version of generated CSV data not synchronize with the guide

### DISCUSSION THREAD

**Discussion by aiya6502** [OHDSI/ETL-Synthea#143] [General]
When I issue the command in R:
TLSyntheaBuilder::LoadSyntheaTables(connectionDetails = cd, syntheaSchema = syntheaSchema, syntheaFileLoc = syntheaFileLoc)

It encountered error:
column "fips" of relation "patients" does not exist when importing CSV data to Synthea
the column "INCOME" generated in patient.CSV is also an extra and will cause issue too.

Any idea to fix it quickly ?

**Comment by fdefalco** [comment]
Which version of Synthea are you using? ETL Synthea currently supports versions 2.7 and 3.0 of Synthea. Neither of those versions contains fips or income.  I suggest you use the 2.7 or 3.0 release of Synthea. You can obtain those releases from the following link:

https://github.com/synthetichealth/synthea/releases
